sympy/vector/coordsysrect.py

```diff
@@ -170,6 +180,16 @@ def __str__(self, printer=None):
     def __iter__(self):
         return iter([self.i, self.j, self.k])
 
+    def __getattr__(self, name):
```

Comment:
I agree, but I thought it'll be better if this is in place before implementing multiple coordinate systems so that it'll be easier to write tests and examples in documentation and make them more understandable.

Shall I close this PR and issue for now?